cmd/juju/controller/killstatus.go

```diff
@@ -99,6 +103,13 @@ func newData(api destroyControllerAPI, controllerModelUUID string) (ctrData, []m
 	var aliveModelCount int
 	var ctrModelData modelData
 	for _, model := range status {
+		if newerAPI && model.Error != nil {
```

Comment:
Why do we need the newerAPI check? Isn't it sufficient to always check for model.Error and you've done the compatibility work in the api/* code so that we'll always have model.Error if we don't know how to proceed?

Reply:
removed.